megamek/src/megamek/client/ui/swing/BoardEditor.java

```diff
@@ -136,69 +226,111 @@ public Component getListCellRendererComponent(JList list, Object value, int inde
             if (-1 < index && null != value && null != terrains) {
                 list.setToolTipText(terrains[index].getTerrainTooltip());
             }
+            if (-1 < index && null != value && null != terrainTypes) {
+                list.setToolTipText(terrainTypes.get(index).getTooltip());
+            }
             return comp;
         }
 
         public void setTerrains(TerrainHelper[] terrains) {
             this.terrains = terrains;
         }
+        
+        public void setTerrainTypess(List<TerrainTypeHelper> terrainTypes) {
```

Comment:
This needs fixed, there's too many 's' here... `setTerrainTypess` should be `setTerrainTypes`

Reply:
Is corrected.